plugin/someteam.example.com/v1/validator/Validator

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+
+# Validator transformer plugin by kubeval
```

Comment:
Maybe add a link to https://github.com/instrumenta/kubeval
is that where you got it?

Reply:
added.